pkg/sql/pgwire/pgerror/errors.proto

```diff
@@ -24,7 +24,8 @@ message Error {
   string detail = 3;
   string hint = 4;
   string severity = 8;
-  string constraint_name = 9;
+  string severity_nonlocalized = 9;
```

Comment:
oh interesting, didn't realize this would need to change

the important thing about `proto` definitions is that the order matters. if you change the order, it changes how the object is serialized, which means code that is using the old definition won't know how to deserialize it

so, you need to make `severity_nonlocalized` be at number 10

Reply:
Just realized this was left over code I meant to remove.